index.js

```diff
@@ -249,6 +249,7 @@ async function fetchSharedPart(firmId, name) {
   } else {
     fetchSharedPartByName(firmId, name);
   }
+  consola.success(`Shared part "${templateConfig.name}" imported`);
```

Comment:
@Woetfin why did you move this here? now is no longer consistent with `fetchReconciliation` for example.

Reply:
For some reason the success message wasn't logging on the previous place, but it was logging here. Reduces duplicate code though so I think it's fine to leave it for now (need to go over all the functions either way for the partner updates, so we can align)